pylearn2/datasets/dataset.py

```diff
@@ -94,14 +171,19 @@ def adjust_for_viewer(self, X):
 
     def has_targets(self):
         """ Returns true if the dataset includes targets """
-
         raise NotImplementedError()
 
     def get_topo_batch_axis(self):
         """
         Returns the index of the axis that corresponds to different examples
         in a batch when using topological_view.
+        WARNING: This method is deprecated and will be unsupported after 27
+                 July 27, 2015. Some classes, e.g. DenseDesignMatrix, might
```

Comment:
Travis doesn't like the fact that you suddenly use a different indentation level here.